domain/builders/atrium-domain-builders-jvm/src/main/kotlin/ch/tutteli/atrium/domain/builders/creating/PathAssertionsBuilder.kt

```diff
@@ -62,4 +63,10 @@ object PathAssertionsBuilder : PathAssertions {
 
     override fun <T : Path> isDirectory(expect: Expect<T>) =
         pathAssertions.isDirectory(expect)
+
+    override fun <T : Path> hasSameTextualContentAs(expect: Expect<T>, targetPath: Path, sourceCharset: Charset, targetCharset: Charset) =
```

Comment:
please add inline (could you please also fix `isRegularFile` and `isDirectory`? thanks)